drivers/sensor/bmi160/bmi160.c

```diff
@@ -18,402 +18,22 @@
-	{16,	BMI160_ACC_RANGE_16G},
-};
-#define BMI160_ACC_RANGE_MAP_SIZE	ARRAY_SIZE(bmi160_acc_range_map)
-
-static const struct bmi160_range bmi160_gyr_range_map[] = {
-	{2000,	BMI160_GYR_RANGE_2000DPS},
-	{1000,	BMI160_GYR_RANGE_1000DPS},
-	{500,	BMI160_GYR_RANGE_500DPS},
-	{250,	BMI160_GYR_RANGE_250DPS},
-	{125,	BMI160_GYR_RANGE_125DPS},
-};
-#define BMI160_GYR_RANGE_MAP_SIZE	ARRAY_SIZE(bmi160_gyr_range_map)
-
-#if defined(CONFIG_BMI160_ACCEL_RANGE_RUNTIME) ||\
-	defined(CONFIG_BMI160_GYRO_RANGE_RUNTIME)
-static int32_t bmi160_range_to_reg_val(uint16_t range,
-				       const struct bmi160_range *range_map,
-				       uint16_t range_map_size)
-{
-	int i;
-
-	for (i = 0; i < range_map_size; i++) {
-		if (range <= range_map[i].range) {
-			return range_map[i].reg_val;
-		}
-	}
-
-	return -EINVAL;
-}
-#endif
-
-static int32_t bmi160_reg_val_to_range(uint8_t reg_val,
-				       const struct bmi160_range *range_map,
-				       uint16_t range_map_size)
-{
-	int i;
-
-	for (i = 0; i < range_map_size; i++) {
-		if (reg_val == range_map[i].reg_val) {
-			return range_map[i].range;
-		}
-	}
-
-	return -EINVAL;
-}
-
-int32_t bmi160_acc_reg_val_to_range(uint8_t reg_val)
-{
-	return bmi160_reg_val_to_range(reg_val, bmi160_acc_range_map,
-				       BMI160_ACC_RANGE_MAP_SIZE);
-}
-
-int32_t bmi160_gyr_reg_val_to_range(uint8_t reg_val)
-{
-	return bmi160_reg_val_to_range(reg_val, bmi160_gyr_range_map,
-				       BMI160_GYR_RANGE_MAP_SIZE);
-}
-
-static int bmi160_do_calibration(const struct device *dev, uint8_t foc_conf)
-{
-	if (bmi160_byte_write(dev, BMI160_REG_FOC_CONF, foc_conf) < 0) {
-		return -EIO;
-	}
-
-	if (bmi160_byte_write(dev, BMI160_REG_CMD, BMI160_CMD_START_FOC) < 0) {
-		return -EIO;
-	}
-
-	k_busy_wait(250000); /* calibration takes a maximum of 250ms */
-
-	return 0;
-}
-
-#if defined(CONFIG_BMI160_ACCEL_RANGE_RUNTIME)
-static int bmi160_acc_range_set(const struct device *dev, int32_t range)
-{
-	struct bmi160_data *data = dev->data;
-	int32_t reg_val = bmi160_range_to_reg_val(range,
-						  bmi160_acc_range_map,
-						  BMI160_ACC_RANGE_MAP_SIZE);
-
-	if (reg_val < 0) {
-		return reg_val;
-	}
-
-	if (bmi160_byte_write(dev, BMI160_REG_ACC_RANGE, reg_val & 0xff) < 0) {
-		return -EIO;
-	}
-
-	data->scale.acc = BMI160_ACC_SCALE(range);
-
-	return 0;
-}
-#endif
-
 #if !defined(CONFIG_BMI160_ACCEL_PMU_SUSPEND)
 /*
  * Accelerometer offset scale, taken from pg. 79, converted to micro m/s^2:
  *	3.9 * 9.80665 * 1000
  */
-#define BMI160_ACC_OFS_LSB		38246
-static int bmi160_acc_ofs_set(const struct device *dev,
-			      enum sensor_channel chan,
+#define BMI160_ACC_OFS_LSB 38246
+static int bmi160_acc_ofs_set(const struct device *dev, enum sensor_channel chan,
 			      const struct sensor_value *ofs)
 {
-	uint8_t reg_addr[] = {
-		BMI160_REG_OFFSET_ACC_X,
-		BMI160_REG_OFFSET_ACC_Y,
-		BMI160_REG_OFFSET_ACC_Z
-	};
+	uint8_t reg_addr[] = { BMI160_REG_OFFSET_ACC_X, BMI160_REG_OFFSET_ACC_Y,
+			       BMI160_REG_OFFSET_ACC_Z };
```

Comment:
These whitespace changes are unrelated to `sensors: split bmi160 to allow an easy wrapping for v2 APIs` and add extra noise to the PR

Reply:
Sorry, I have clang-format to auto-run on saving and I missed that. Most of the time the files are already properly formatted and it's not an issue.